src/Resources/skeleton/doctrine/Repository.tpl.php

```diff
@@ -21,7 +21,9 @@ public function __construct(ManagerRegistry $registry)
     {
         parent::__construct($registry, <?= $entity_class_name; ?>::class);
     }
+<?php if (false === $avoid_default_comments) { // When adding a new method without existing default comments, the blank line is automatically added.?>
```

Comment:
2 minor things:

1) When you update this to be `include_example_comments,` just make the if statement ` if ($include_example_comments)` - no need to check `=== true`. I admit that this is something I've been going back and forth on recently, but I've gotten advice from a smart friend to keep these if statements simpler.

2) When you have if statements in a template like this, use this syntax:

```php
<?php if ($include_example_comments): ?>

<?php endif; ?>
```

Reply:
Done.